lua/telescope/make_entry.lua

```diff
@@ -890,12 +891,10 @@ function make_entry.gen_from_lsp_diagnostics(opts)
       string.format("%s %s", signs[entry.type], pos),
       string.format("LspDiagnosticsDefault%s", entry.type)
     }
-    -- remove line break to avoid display issues
-    local text = string.format("%-" .. line_width .."s", entry.text:gsub(".* | ", ""):gsub("[\n]", ""))
 
     return displayer {
       line_info,
-      text,
+      entry.text,
```

Comment:
Is there still a chance where we get `\n` in display? Because i also ran into this a couple of weeks ago and this breaks things. So we can just do `gsub` just in case.